### PR TITLE
fix output messages

### PR DESCRIPTION
**Pull request by floweisshardt**
- fix detector name to reflect nodename (necessary for multiple detector setup)
- fill empty headers with content (stamp and frame_id) from input message

@ipa-rmb: please review

### CODE REVIEW

File: cob_people_detection/ros/src/face_recognizer_node.cpp [hunk 1]
```diff
@@ -518,7 +518,7 @@ void FaceRecognizerNode::facePositionsCallback(const cob_perception_msgs::ColorD
 				// set label
 				det.label = identification_labels[head][face];
 				// set origin of detection
-				det.detector = "face";
+				det.detector = ros::this_node::getName();
```
Comment (ipa-rmb) [inline cob_people_detection/ros/src/face_recognizer_node.cpp]: I doubt that the detection tracker will work with these names without further modification. I am pretty sure that the tracking distinguishes detections by their type between head and face for updating confidences. Hence, add a fix for the detection tracker as well.

E.g. in https://github.com/ipa320/cob_people_perception/blob/indigo_dev/cob_people_detection/ros/src/detection_tracker_node.cpp#L450

Comment (floweisshardt) [inline cob_people_detection/ros/src/face_recognizer_node.cpp]: mhh, ok.

there are three fields in the message that are unclear to me:
- detector
- label
- id

Can you explain for what they are used?

I'm searching for a field that allows me to distinguish where a messages comes from in a multi detector setup.

Comment (ipa-rmb) [inline cob_people_detection/ros/src/face_recognizer_node.cpp]: label is the name of the identified person, I have no idea about id, is it set by people detection? The message format is inherited from object detection, so id might be free

However, detector is the field that is intended to be used as a clue to the source. What does ros::this_node::getName() yield? You may also put into the detector field: "head," + ros::this_node::getName() and later adapt the code in the detection tracker to search whether detector contains/begins with "head" or "face" and still use the remaining information of the string elsewhere.

Comment (floweisshardt) [inline cob_people_detection/ros/src/face_recognizer_node.cpp]: ok, reverted changes on `detector`, using `frame_id` for fusion node

**Comment by floweisshardt** [comment]
@ipa-rmb: ready for merge?

**Comment by ipa-rmb** [comment]
Looks ok.

**Pull request merged**